framework/SwiftPlot/ScatterChart.swift

```diff
@@ -0,0 +1,378 @@
+import Foundation
+
+// class defining a lineGraph and all its logic
+public class ScatterPlot: Plot {
+
+    let MAX_DIV: Float = 50
+
+    let sqrt3: Float = sqrt(3)
+
+    public var xOffset: Float = 0
+    public var yOffset: Float = 0
+
+    public var plotTitle: PlotTitle = PlotTitle()
```

Comment:
This may apply elsewhere, but thinking about it, would it make sense to have plotTitle, etc. be optionals? That way, they can be selectively disabled when not needed by setting the values to nil.

You could use `if let` logic later to control the drawing or not of each plot feature. This may require a rework of other plots to match, but it would seem to be a clean way of selectively disabling plot elements.